### DISCUSSION TITLE
No folder named Bin64 (xbox store version) - different files/folders

### DISCUSSION THREAD

**Discussion by wiwi012** [Lamer87/Space_Engineers_DLC_unlocker#47] [Q&A] [answered]
If a make one, the file still doesn't work.

![image](https://github.com/Lamer87/Space_Engineers_DLC_unlocker/assets/123038065/30344d26-fd82-4ca2-9d11-415b0e3ef8dd)

I downloaded game through xbox store.

**Comment by littleAndyy** [comment, marked as answer]
it might be PC / steam only for Bin64, but I don't 100% know for sure :(

**Comment by Lamer87** [comment]
> I downloaded game through xbox store.

Maybe this is the problem, sorry but I don't know how to help you in this case.
